README.md

```diff
@@ -1 +1,91 @@
-# My-Portfolio
\ No newline at end of file
+# My-Portfolio
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+# 📖 [Set-up-mobile-first-portfolio] <a name="about-project"></a>
+
+**[Set-up-mobile-first-portfolio]** is a project in which we use our knowledge about html and css also the knowledge about linters.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+I used VS code to create the project and files needed plus used Git and GitHub to complete the tasks. I used templates as well.
+
+### Key Features <a name="key-features"></a>
+
+- **[It will show my portfolio website made by using css and html.]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+It will show my sample portfolio.
+
+### Prerequisites
+
+In order to run this project you need to clone the repo using git bash and run the html file in browser using VS code.
```

Comment:
- [ ] It will be 😃better if you write descriptive info about how to use the project like to git clone the project ``` git clone here add the path of repo``` and you can add the npm i to see dependencies of the project...